spring-app/src/main/java/pl/edu/agh/cqm/controller/MonitorController.java

```diff
@@ -0,0 +1,31 @@
+package pl.edu.agh.cqm.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import pl.edu.agh.cqm.data.dto.MonitorsDTO;
+import pl.edu.agh.cqm.data.dto.MonitorsResponseDTO;
+import pl.edu.agh.cqm.data.repository.MonitorRepository;
+import pl.edu.agh.cqm.service.MonitorService;
+
+import javax.validation.Valid;
+
+@CrossOrigin
+@RestController
+@RequestMapping("api/monitors")
+@AllArgsConstructor
+public class MonitorController {
+
+    private final MonitorService monitorService;
+
+    @PutMapping
+    public void put(@Valid @RequestBody MonitorsDTO monitorsDTO) {
+
+    }
```

Comment:
monitors will be created when submitting samples, this endpoint is probably not needed